charts/jenkins-jobs/templates/_helpers.tpl

```diff
@@ -130,12 +130,17 @@ multibranchPipelineJob('{{ .fullId | default .id }}') {
           traits {
             gitHubSCMSourceStatusChecksTrait {
               // Note: changing this name might have impact on github branch protections if they specify status names
-              name('[infra.ci.jenkins.io] {{ .name }}')
-              skip({{ .disableGitHubChecks | default "false" }})
+              name({{ .githubCheckName | default "jenkins" | squote }})
+              {{- if empty .enableGitHubChecks }}
```

Comment:
Testing `if .enableGitHubChecks` isn't enought here I suppose? Would have been simplier than `skip({{ not .enableGitHubChecks }})`

Reply:
- In term of expctation, if you want to enable the github checks, then you want `skip(false)` in the generated job-dsl config => that is the reason of the `not`
- And `{{- if .enableGitHubChecks }}` fails if the attribute is unspecified (like "parsing error").
- And of course `{{- if or (empty .enableGitHubChecks) (not .enableGitHubChecks) }}` also fails if the attribute is unspecified